app/views/lists/public_show.html.erb

```diff
@@ -2,7 +2,7 @@
 <br>
 
   <div id ="public_search_loop">
-    <%= render partial: 'movies/movie_partial_loop', object: @movies %>
+    <%= render partial: 'movies/movie_partial_loop', locals: { movies: @movies } %>
```

Comment:
To ensure expected behavior, I switched the way we call this partial in every file to ensure we're using local variables.